docs/_documentations/mdt-eclipse-installinfo.md

```diff
@@ -23,12 +23,10 @@ The following images are pulled. These images together form the Codewind back en
 - `eclipse/codewind-pfe-amd64`
 
 ## Using Codewind
-When the installation is complete, the extension is ready to use, and you are prompted to open the Codewind workspace.
-
-You can open the `codewind-workspace` or a project within the workspace as your Eclipse workspace. For more information, see [Getting started: Codewind for Eclipse](mdteclipsegettingstarted.html).
+When the installation is complete, the extension is ready to use.
 
 ## Working with workspaces
-Codewind creates the Codewind workspace on your system. The Codewind workspace is where Codewind expects to find your projects and where Codewind stores its metadata. On macOS and Linux, the workspace folder is `~/codewind-workspace`. On Windows, the workspace folder is `C:\codewind-workspace`. When Codewind creates a new project, it is created in the Codewind workspace. If you want to add an existing project to Codewind, first copy it into the Codewind workspace.
+Codewind creates a codewind-data directory for storing its metadata. You should not save your projects to the codewind-data directory. 
```

Comment:
"should not" -> "must not"

Reply:
I think "Do not" works here, too.
Ex: "Do not save..."